data/json/mutations/mutations.json

```
@@ -8039,7 +8045,7 @@
     "category": [ "PLANT" ],
     "scent_type": "sc_flower",
     "encumbrance_covered": [ [ "foot_l", 10 ], [ "foot_r", 10 ] ],
-    "flags": [ "TOUGH_FEET", "ROOTS3" ]
+    "flags": [ "TOUGH_FEET", "ROOTS3", "IRRITANT_IMMUNE" ]
```

```suggestion
    "flags": [ "TOUGH_FEET", "ROOTS3" ]
```
Chlorophyll doesn't make plants more or less immune to products targeting it's cells.

---

Yeah, you're right.